BrainPortal/app/views/portal/_available_tools_table.html.erb

```diff
@@ -53,7 +53,14 @@
     end
   %>
   <tr>
-    <td><%= overlay_description((name + "\n" + description).strip) %></td>
+    <td>
```

Comment:
Same here. Like I says, look at the original code you replaced, and ask yourself why is the new code more complicated? That should be an immediate red flag in your mind.